docs/integration.rst

```diff
@@ -510,6 +510,14 @@ Compute Engine Operators
 - :ref:`GceInstanceStartOperator` : start an existing Google Compute Engine instance.
 - :ref:`GceInstanceStopOperator` : stop an existing Google Compute Engine instance.
 - :ref:`GceSetMachineTypeOperator` : change the machine type for a stopped instance.
+- :ref:`GceInstanceTemplateCopyOperator` : copies the instance template, applying specified changes
+- :ref:`GceInstanceGroupManagerUpdateTemplateOperator` : Patches the Instance Group Manager, replacing source template URL with the destination one.
```

Comment:
Consistency with regards to capital / small letters and dots at EOL.

Reply:
- `copies the instance template, applying specified changes` should have a dot at the end.
- `Patches the Instance Group Manager, replacing source template URL with the destination one.` should begin with a lowercase `patches`.

Reply:
Fixed. I rebased to the latest master as well.